apps/src/templates/projects/ProjectsGallery.jsx

```diff
@@ -51,11 +51,19 @@ class ProjectsGallery extends Component {
           {galleryTabs.map(tab => (
             <div
               key={tab.key}
+              tabIndex="0"
+              role="button"
               style={[
                 styles.pill,
                 this.props.selectedGallery === tab.key && styles.selectedPill,
               ]}
               onClick={() => this.toggleTo(tab)}
+              onKeyDown={e => {
```

Comment:
For my own learning- can you help me understand how the different keys work here? Is this a way for 'tab' to move through, and 'spacebar' or 'enter' to effectively 'select'?

Reply:
You got it! `onKeyDown` is just an event handler that let's you specify what should happen when certain keys are pressed. In this case, when space and enter are pressed, we effectively want to "click" on that tab to navigate that selected tab's content.

Reply:
Excellent. I should've just looked into the definition of 'toggleTo' to confirm!